src/tools/rustfmt/src/items.rs

```diff
@@ -666,7 +666,7 @@ impl<'a> FmtVisitor<'a> {
         let span = mk_sp(lo, field.span.lo());
 
         let variant_body = match field.data {
-            ast::VariantData::Tuple(..) | ast::VariantData::Struct(..) => format_struct(
+            ast::VariantData::Tuple(..) | ast::VariantData::Struct{..} => format_struct(
```

Comment:
```suggestion
            ast::VariantData::Tuple(..) | ast::VariantData::Struct { .. } => format_struct(
```

Reply:
Weird that the tidy pre-push hook didn't catch this.